samples/config/recover_config.json

```
@@ -24,7 +24,8 @@
     "type": "Recover",
     "service_certificate_file": "service_cert.pem",
     "recover": {
-      "initial_service_certificate_validity_days": 1
+      "initial_service_certificate_validity_days": 1,
+      "previous_service_identity_file": "/tmp/previous_service_cert.pem"
```

```suggestion
      "previous_service_identity_file": "previous_service_cert.pem"
```

To be consistent with other paths?